### PR TITLE
test: increase coverage

### PR DESCRIPTION
**Pull request by cristianossd**
Related to #29 

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

#### Coverage report

- Before
```
----------|---------|----------|---------|---------|-------------------
File      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
----------|---------|----------|---------|---------|-------------------
All files |   94.59 |    73.91 |     100 |   93.94 |                   
 flash.ts |   96.15 |       80 |     100 |   95.45 | 19                
 index.ts |   90.91 |    33.33 |     100 |   90.91 | 13                
----------|---------|----------|---------|---------|-------------------
```

- After
```
----------|---------|----------|---------|---------|-------------------
File      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
----------|---------|----------|---------|---------|-------------------
All files |     100 |    82.61 |     100 |     100 |
 flash.ts |     100 |       85 |     100 |     100 | 48-51
 index.ts |     100 |    66.67 |     100 |     100 | 5
----------|---------|----------|---------|---------|-------------------
```

### CODE REVIEW

**Comment by climba03003** [comment]
Can you provide the coverage report? If it reach 100% in all fields, can you remove this line https://github.com/fastify/fastify-flash/blob/master/.taprc#L5.

**Comment by cristianossd** [comment]
Of course, @climba03003. I added in description

**Review by mcollina — APPROVED** [review]
lgtm

**Review by climba03003 — APPROVED** [review]
LGTM.

File: tests/index.ts [hunk 1]
```diff
@@ -239,3 +239,39 @@ test('should format messages.', async t => {
   t.equal(response.payload, '{"jared":"Hello Jared","jaredHanson":"Hello Jared Hanson"}')
   t.equal(response.statusCode, 200)
 })
+
+test('should return error when session is not defined.', async t => {
```
Comment (Eomm) [inline tests/index.ts:243]: This test let me think that should we trigger a startup error instead of a runtime one?

Comment (climba03003) [reply tests/index.ts:243]: Yes, it is better to throw an error in startup as `session` is a must for this plugin.

Comment (cristianossd) [reply tests/index.ts:243]: Is it the case to create an issue for?

**Pull request merged**